pkg/services/ngalert/notifier/redis_peer.go

```diff
@@ -100,8 +99,9 @@ func newRedisPeer(cfg redisConfig, logger log.Logger, reg prometheus.Registerer,
 	})
 	cmd := rdb.Ping(context.Background())
 	if cmd.Err() != nil {
-		return nil, fmt.Errorf("failed to ping redis: %w", cmd.Err())
+		logger.Error("Failed to ping redis - redis-based alertmanager clustering may not be available", "err", cmd.Err())
 	}
```

Comment:
[nit] I wonder if we just delete this code entirely? We will immediately start getting errors from the `heartbeatLoop` anyway. I'm happy either way, I'll leave it to your judgement.